manage/sawtooth_manage/vnm.py

```diff
@@ -23,8 +26,21 @@ def __init__(self,
 
     def update(self):
         commands = self._node_command_generator.get_commands()
+
+        # execute commands in parallel
+        threads = []
         for command in commands:
-            command.execute(self._node_controller)
+            thread = Thread(
+                target=command.execute,
+                args=(self._node_controller,))
+            thread.start()
+            threads.append(thread)
+            time.sleep(1)
+
+        # block until all threads have completed or timed out
+        wait_time = 10
+        for thread in threads:
+            thread.join(wait_time)
```

Comment:
If this times out, are there any negative impacts (such as the command not stopping)?